docs/shrc_diag_util.md

```diff
@@ -262,106 +282,170 @@ Line number information for classes in the cache is then shown with one of the f
 
 The following summary data is displayed:
 
-#### `baseAddress` and `endAddress`
- : The boundary addresses of the shared memory area that contains the classes.
-
-#### `allocation pointer`
- : The address where `ROMClass` data is currently being allocated in the cache.
-
-#### `cache layer`
-: The layer number that the cache stats relate to.
-
-#### `cache size` and `free bytes`
- : `cache size` shows the total size of the shared memory area in bytes, and `free bytes` shows the free bytes that remain.
+- `baseAddress` and `endAddress`
+    - The boundary addresses of the shared memory area that contains the classes.
+- `allocation pointer`
+    - The address where `ROMClass` data is currently being allocated in the cache.
+- `cache layer`
+    - The layer number that the cache stats relate to.
+- `cache size` and `free bytes`
+    - `cache size` shows the total size of the shared memory area in bytes
+    - `free bytes` shows the free bytes that remain.
+- `softmx bytes`
+    - The soft maximum size for the cache. For more information, see [`-Xscmx`](xscmx.md).
+- `ROMClass bytes`
+    - The number of bytes of class data in the cache.
```

Comment:
Not including the class debug data, which is typically stored separately and shown by the Class debug area summary data.